app/assets/stylesheets/pageflow/page_transitions/scroll_over_from_right.scss

```diff
@@ -0,0 +1,12 @@
+section.scroll_over_from_right {
+  &.animate-in-forwards, &.animate-out-backwards {
+    z-index: 2;
+  }
+
+  &.animate-in-forwards {
+    @include animation(scroll-in-right forwards 1s 1);
+  }
+  &.animate-out-backwards {
```

Comment:
Selector `animate-out-backwards` should be written in lowercase with underscores